arm/cookbooks/win_workstation/test/recipes/default_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'inspec'
+
+describe package 'git' do
```

Comment:
That's a tricky question:

- Since chefspec tests are assertions against a compiled Chef object list, they can run _a lot_ faster than test-kitchen with a virtualized Win2012R2 instance
- ChefSpec also can let you test your code logic against a huge test matrix, so you ask, does my code do the right thing w/ a box with 512Gb of RAM? 
- In general, though, I've used ChefSpec just as a fail-fast test before doing test-kitchen, since I don't want to wait 5m to find out I wrote correct Ruby syntax, but failed Chef logic. So writing all the package tests in ChefSpec is not really a useful exercise
- and in the end, test-kitchen is more useful than chefspec for assuring your code is going to do what you want it to do.

HTH - if you have questions still we should talk it over.